src/integrationTest/codewhisperer/util.ts

```diff
@@ -0,0 +1,36 @@
+/*!
```

Comment:
Test util code should live in `src/integrationTest/util/`. That makes it clear the total size of the "test util" code, which should be minimal.

In fact, much of what's here already looks potentially generalizable for non-codewhisperer tests in the future.

Reply:
Thanks for your comment. That makes sense. I moved the `util` file to `src/integrationTest/util/`.

I also renamed `util` to `codewhispererUtil` to avoid confusion since it is only being used for codewhisperer auth connection now, but to your point this / the related imports can be easily updated if this logic is generalized or updated for other tests in the future.